lib/client-assets.php

```diff
@@ -653,8 +653,17 @@ function gutenberg_default_script_modules() {
 				break;
 		}
 
+		/*
+		 * All script modules in Gutenberg are (currently) related to the Interactivity API which prioritizes server-side rendering.
```

Comment:
There's `@wordpress/a11y` module that's not strictly an iAPI concept.

https://make.wordpress.org/core/2024/10/14/updates-to-script-modules-in-6-7/

I don't _think_ other modules have been introduced, but I could have missed something. Should `a11y` be `low` as well?